gflownet/policy/base.py

```diff
@@ -2,12 +2,11 @@
 
 import torch
 from omegaconf import OmegaConf
-from torch import nn
 
 from gflownet.utils.common import set_device, set_float_precision
 
 
-class ModelBase(ABC):
+class Policy(ABC):
```

Comment:
It's not very crucial, but I'd suggest to call it `BasePolicy` to highlight that it is the base class to inherit from. Also, I think it's not needed anymore to inherit from `ABC` here as all the `@abstractmethod ` decorators were removed

Reply:
This sounds good, will resolve in next commit